### PR TITLE
logo-ls: init at 1.3.7

### LINKED ISSUE #229175: Package request: Logo-Ls
modern ls command with beautiful Icons and Git Integrations . Written in Golang

* source URL: https://github.com/Yash-Handa/logo-ls
* license: mit, bsd, gpl2+ , ...
* platforms: unix, linux, darwin, ...

### PR DESCRIPTION
**Pull request by natsukium**
###### Description of changes

Modern ls command with vscode like File Icon and Git Integrations.
https://github.com/Yash-Handa/logo-ls

closes #229175 

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/logo-ls/default.nix [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    installManPage logo-ls.1.gz
+  '';
+
+  meta = with lib; {
+    description = "Modern ls command with vscode like File Icon and Git Integrations";
+    homepage = "https://github.com/Yash-Handa/logo-ls";
+    changelog = "https://github.com/Yash-Handa/logo-ls/releases/tag/v${version}";
+    license = licenses.mit;
+    maintainers = with maintainers; [ natsukium ];
+    platforms = platforms.unix;
```
Comment (figsoda) [inline pkgs/tools/misc/logo-ls/default.nix:34]:
```suggestion
```

File: pkgs/tools/misc/logo-ls/default.nix [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
```
Comment (figsoda) [inline pkgs/tools/misc/logo-ls/default.nix:3]:
```suggestion
, buildGoModule
, fetchFromGitHub
```

**Review by figsoda — COMMENTED** [review]
tests are failing on aarch64, could you take a look and maybe skip them or mark it as broken?

File: pkgs/tools/misc/logo-ls/default.nix (hunk 1 above)
Comment (natsukium) [reply pkgs/tools/misc/logo-ls/default.nix:34]: Thanks for checking.

It is due to a filename conflict, as shown in this issue.
https://github.com/Yash-Handa/logo-ls/issues/26
I will create a patch to fix it.

**Pull request closed without merging**